.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,14 @@
+name: auto-merge
+
+on:
+  pull_request:
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
+          github-token: ${{ secrets.AUTO_MERGE_DEPENDBOT }}
```

Comment:
Is this a typo?
```suggestion
          github-token: ${{ secrets.AUTO_MERGE_DEPENDABOT }}
```

Reply:
shoot it is, but I copied and pasted the token name :facepalm: so it "works" but I don't think I can rename the token without having to remake it, I'll fix it even if I have to remake

Reply:
okay fixed the name in the code and also the token name in our secrets page